docs/demonstration/canvas-e2e.ipynb

```diff
@@ -16,7 +16,7 @@
     "## Getting Started\n",
     "\n",
     "1. Confirm you have [poetry](https://python-poetry.org) installed (`poetry --version`).\n",
-    "1. Follow the [notebook instructions](README.md) to install dependencies used by this notebook.\n",
+    "1. Follow the [notebook instructions](../../src/canvas-extractor/README.md) to install dependencies used by this notebook.\n",
```

Comment:
I don't think this change is correct.

Reply:
For LMS-480: there are [dependencies](https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit/blob/main/src/notebooks/pyproject.toml) for the notebook folder itself, so the original line was correct - you need to install the dependencies for the notebook folder. Think about reading this file in GitHub or in VS Code - you can be reading it without having launched Jupyter Notebook.

But I think you're right that another line is needed: a reminder to install the extractor's dependencies.